index.html

```diff
@@ -25,9 +25,223 @@
             <input type="checkbox" class="checkbox" value="log-in">
             <p class="left">Keep me logged in</p>
             <a href="https://www.facebook.com/login/identify?ctx=recover"><p class="right">Forgot your password?</p></a>
+
           </div>
         </form>  
       </div>   
     </nav>
+    <section>
+      <div class="main-section">
+        <div class="left-wrapper">
+          <div>
+            <h1>Connect with your friends and the world around you on Facebook.</h1>
+          </div>
+          <div class="top">
+            <img src="section-newspaper.png">
+           <h2 class="top-text"><span>See photos and updates</span> from friends in News Feed.</h2>
+          </div>
+          <div class="middle">
+            <img src="section-tv.png">
+            <h2 class="middle-text"><span>Share what's new</span> in your life on your Timeline.</h2>
+          </div>
+          <div class="bottom">
+            <img src="section-share.png">
+            <h2 class="bottom-text"><span>Find more</span> of what you're looking for with Graph Search.</h2>
+          </div>    
+        </div>
+        <div class="right-wrapper">
+          <h2>Sign Up</h2>
+          <h3>It's free and always will be.</h3>
+          <div class="signup-form">
+            <form method="post" action="https://m.facebook.com/r.php">
```

Comment:
Don't use the same action as FB. Just use `action="#"` for the moment. That way your page won't redirect to FB.